package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "outsystems-plugin-fingerprint",
+  "version": "2.0.1",
+  "description": "Cordova FingerPrint Plugin",
+  "cordova": {
+    "id": "outsystems-plugin-fingerprint",
+    "platforms": []
+  },
+  "dependencies": {
+    "cordova-plugin-android-fingerprint-auth":"https://github.com/OutSystems/cordova-plugin-android-fingerprint-auth.git#0.3.0-OS",
```

Comment:
I'm not sure if plugin dependencies are to be defined on package.json or not, can't recall an instance where I've seen this happening, usually they are defined on plugin.xml alone 🤔